src/pages/workspaces/workspace/Workflows.js

```diff
@@ -104,13 +123,20 @@ class Workflows extends Component {
 
   renderSubmissions() {
     const { namespace } = this.props
-    const { submissions, loading } = this.state
+    const { submissions, loading, newSubmissionId } = this.state
 
     return div({ style: styles.submissionsTable }, [
       loading && spinnerOverlay,
       submissions && h(AutoSizer, [
         ({ width, height }) => h(FlexTable, {
           width, height, rowCount: submissions.length,
+          rowStyle: rowIndex => {
+            const { submissionId } = submissions[rowIndex]
+            return {
+              transition: 'all 3s linear',
```

Comment:
nit: I think it's preferable to be explicit about what properties you want to enable transitions for, rather than using `all`